starenchenkonv/first/task2.py

```diff
@@ -0,0 +1,7 @@
+text = "hello my Name Is Nazar"
+
+text_list = text.split()
+
+for word in text_list:
+    if word[0].isupper():
+        print(word)
```

Comment:
text = "hello my Name Is Nazar"


def get_all_upper_starts_words(text):
    words_list = []
    text_list = text.split()
    for word in text_list:
        if word[0].isupper():
            words_list.append(word)
            print(word)
    return words_list


get_all_upper_starts_words(text)